platform/lf_nrf52_support.c

```diff
@@ -104,6 +111,18 @@ void calculate_epoch_offset() {
  */
 uint32_t _lf_previous_timer_time = 0u;
 
+void lf_timer_event_handler(nrf_timer_event_t event_type, void *p_context)
```

Comment:
Function comment needed. What does this do?

Reply:
added